filestore/test/test_write.py

```diff
@@ -65,14 +64,14 @@
 conn = None
 
 
-def setup():
+def setup_module(module):
     fs_setup()
```

Comment:
AFAICT, fixtures are generally used to pass in objects that tests can manipulate.  fs_setup doesn't return anything and so does not map onto the fixture model well.  

e.g., 

```python
# content of ./test_smtpsimple.py
import pytest

@pytest.fixture
def smtp():
    import smtplib
    return smtplib.SMTP("smtp.gmail.com")

def test_ehlo(smtp):
    response, msg = smtp.ehlo()
    assert response == 250
    assert 0 # for demo purposes
```

The above example, copied from the pytest docs shows the return value of `smtp` being used by `test_ehlo`.  The fs_setup and fs_teardown don't seem to map onto such a model.

I could see a valid argument for for making this be `setup_function` and `teardown_function` instead of setup_module and `teardown_module` for reasons of better test isolation.